### PR TITLE
Edit Logo for Table of Content

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
Adding my esports organization (**ArchEsports**) logo in the **_img_** folder and replacing the image in the README.md file 😄

### CODE REVIEW

**Comment by JuanDiegoGuerra** [comment]
The **Stylelint Linter** request seems to be a sort of bug cause I dont know why it says: _"Expected indentation of 2 spaces"_
and the **Lighthouse** one is always waiting for browser and I don't know what that means.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <a name="readme-top"></a>
 
 <div align="center">
-  <img src="img/fbprofilepic.jpg" alt="logo" width="50"  height="auto" />
+  <img src="img/archesportslogo.jpg" alt="logo" width="50"  height="auto" />
   <br/>
 </div>
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Nice work adding some of the sections on the [README](https://github.com/microverseinc/readme-template), however, you're meant to add all sections with just `FAQ` being optional. I've attached screenshots below to show you what to add. This makes your project possess all sections that make it look professional 

## Expected README  Header:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @JuanDiegoGuerra       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper use of .gitgnore
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below needs to be implemented:

- Please make sure all linters are passing
- Your `MIT` License seems to be broken, kindly fix this 👍 
- I highlighted some sections of your README to be added. Kindly check the comments and [Template](https://github.com/microverseinc/readme-template) provided

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by JuanDiegoGuerra** [comment]
@Shedrack-Sunday Hello! I just finished everything 🤩. It was kinda hard working for me. It wasn't hard at all but I think there was a lot of information I had to review in order to complete it. Hope I fixed everything you tell me to. I'll be waiting for your feedback. Thanks again!
👉🏼All Linters are passing: ✅
Success
♻Fixing the LICENSE (it wasn't there for some reason.) ✅
LICENSE
✏Last commit to add all the information needed in the README file: ✅
UpdateREADME

**Review by Shedrack-Sunday — APPROVED** [review]
# STATUS: APPROVED ♻️♻️
Hi @JuanDiegoGuerra 

To highlight 👍🏽
✔️ Proper use of semantics
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by shella12 — COMMENTED** [review]
Hi @JuanDiegoGuerra , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has wrong project link. You are submitting" Hello Microverse" project link for "portfolio" project. Please make sure to submit the correct link on to dashboard.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**